projects/aca-content/src/lib/components/details/details.component.scss

```diff
@@ -5,6 +5,12 @@ app-details-manager {
     outline: none;
     border-radius: 4px;
 
+    &:focus {
+      background-color: #1f74db3d;
```

Comment:
hardcoded color, should be coming from theme variables

Reply:
Updated  with theme varaibles